src/styles/index.scss

```diff
@@ -1,3 +1,136 @@
+:root {
+  --link-color: #00acdc;
+  --color: #616070;
+  --bg-color: #fff;
+  --text-color: #060b35;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__image img {
+  margin: 0 auto;
+  max-width: 160px;
+  display: block;
+  height: 100%;
+}
+
+.card__text {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--text-color);
```

Comment:
Font sizes and line heights should use relative units instead of 'px' to improve scalability and accessibility. Refer to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) for best practices on using relative units like 'em' or 'rem'.